module/VuFind/src/VuFind/ILS/Driver/Demo.php

```diff
@@ -1530,7 +1530,7 @@ function ($loc) use ($excluded) {
      * @param array $holdInfo Contains most of the same values passed to
      * placeHold, minus the patron data.
      *
-     * @return int
+     * @return int|null
```

Comment:
`?int` might be more concise here (though I suspect you copied this from Aleph, so if you prefer to keep it consistent with that, I have no objection).

Reply:
I did this since @EreMaijala suggested to standardize it here: https://github.com/vufind-org/vufind/pull/3112#discussion_r1335444249

It looks like it is done like this in a lot of places. So, I don't think this this violates any code style rules. But if you prefer `?int` I can change this.

Reply:
Feel free to leave it alone for now -- since it's consistent with other things, there's no need to change it. I think in the long term I'd prefer the `?` syntax instead of the `|null` syntax since it is more concise -- but fixing that should probably become a separate project, and it's best to back it up with some kind of style checker if we make that decision. All out of scope for what you're doing here. :-)

Reply:
I didn't mean to imply that you should use `int|null`as `?int` is indeed better, though I also value keeping the headers of same function consistent. Perhaps we could also add more interfaces and use `@inheritDoc` to avoid copy-pasting and ensure that the method definitions don't diverge.

Reply:
The problem with inheritDoc is that php codesniffer doesn't support it properly, so we'd either need to disable comment validation or ignore lots of errors. I wish this would get addressed, but I think it's a very hard problem to solve based on the way codesniffer works.